client/src/index.jsx

```diff
@@ -1,6 +1,26 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {
+  Redirect,
+  BrowserRouter as Router,
+  Route,
+  Switch
+} from 'react-router-dom';
 
 import App from './components/App';
 
-ReactDOM.render(<App />, document.getElementById('app'));
+const DEFAULT_ROOM_ID = '11111111';
+
+ReactDOM.render(
+  <Router>
+    <Switch>
+      <Route path={`/rooms/:roomId`}>
```

Comment:
(line 17) change backticks here to single quotes as you're not using a template literal

Reply:
@lndncole Fixed 😄